### PR TITLE
Fix invalid "ls -G" alias.

### PR DESCRIPTION
**Pull request by reduardo7**

### CODE REVIEW

**Comment by xuhdev** [comment]
Looks like you need to rebase now...

**Comment by mcornella** [comment]
@reduardo7 what does this solve? What's your setup? (platform, operating system, installed ls's...)

**Comment by reduardo7** [comment]
@mcornella this solve a problem with ZSH in Termux (https://play.google.com/store/apps/details?id=com.termux). You can try install it and configure with ZSH + oh-my-zsh. When you type `ll`, results in an error.

**Comment by red15** [comment]
Causes problems for other users #5604

File: lib/theme-and-appearance.zsh [hunk 1]
```diff
@@ -25,7 +25,7 @@ if [[ "$DISABLE_LS_COLORS" != "true" ]]; then
       (( $+commands[dircolors] )) && eval "$(dircolors -b)"
     fi
 
-    ls --color -d . &>/dev/null && alias ls='ls --color=tty' || alias ls='ls -G'
+    ls --color -d . &>/dev/null && alias ls='ls --color=tty' || ls -G . &>/dev/null && alias ls='ls -G'
```
Comment (croachrose) [inline lib/theme-and-appearance.zsh]: This should be:

`ls --color -d . &>/dev/null && alias ls='ls --color=tty' || (ls -G . &>/dev/null && alias ls='ls -G')`

Note the parenthesis around the second test. This should fix the various ls colors issue that people are seeing.

**Pull request closed without merging**